src/Tribe/Customizer.php

```diff
@@ -433,7 +433,7 @@ public function widget_inline_style() {
 	/**
 	 * Print the CSS for the customizer using wp_add_inline_style
 	 *
-	 * @since TBD Added the `$force` parameter to force the print of the style inline.
+	 * @since 4.12.15 Added the `$force` parameter to force the print of the style inline.
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Line exceeds 80 characters; contains 90 characters

<sub>hash: 3538874eb355aa89b9e7a60d88ec1aa8</sub>